test/kind/Makefile

```diff
@@ -16,3 +16,37 @@ dev:
 	docker push localhost:5000/calicovpp/agent:$(TAG)
 	docker image tag calicovpp/multinet-monitor:$(TAG) localhost:5000/calicovpp/multinet-monitor:$(TAG)
 	docker push localhost:5000/calicovpp/multinet-monitor:$(TAG)
+
+
+.PHONY: load
+load:
+	docker pull docker.io/calico/node:master
+	docker save docker.io/calico/node:master -o /home/hbouatto/images/node
```

Comment:
We could probably either `mkdir $SCRIPTDIR/.imagecache/` and use this as a cache directory

Reply:
sorry for that personal home 😂

Reply:
😄 np, did the same more than a couple times